src/Tools.js

```diff
@@ -12,15 +12,15 @@ class Tools {
 	 * within a bidi isolation block.
 	 *
 	 * @param {jQuery|string} $content Content to isolate
-	 * @param {boolean} returnRawHtml Force the return value to
+	 * @param {boolean} [returnRawHtml] Force the return value to
 	 *  be raw html. If set to false, will return the encompassed
 	 *  jQuery object.
 	 * @return {jQuery|string} BiDi isolated jQuery object or HTML
 	 */
-	static bidiIsolate( $content, returnRawHtml ) {
+	static bidiIsolate( $content, returnRawHtml = false ) {
 		const $result = $( '<bdi>' );
 		if ( typeof $content === 'string' ) {
-			$result.text( $content );
+			$result.html( $content );
```

Comment:
@mooeypoo I had to change this to leave HTML unescaped because the API sometimes gives us markup (for section or other internal links). The API escapes any other markup, so I think we're safe? Though I realize this function is probably meant to be used on anything, and not just edit summaries. Is there a better to do this?

Reply:
So the point of this is that the tool is expecting the user of the tool to do the safety check for it. If it's a string, it will be treated as such, on purpose. We do something similar in mw.msg internals.

This forces the use case that calls this function to intentionally encapsulate. So, the best way to do it is wrap the response in a jQuery object before sending it to this utility. 

It's mostly a conceptual way to make sure that when we read the code in x years, we see very clearly that whatever was (and wasn't) escaped was done purposefully, rather than trying to convince ourselves all over again if it's safe.

Does that make sense? I'd just encapsulate in jQuery before sending here. 

We do this in ooui `setLabel` too, fly the same reasons.

Reply:
Done! Except I did it in `Tools.bidiIsolate()`, which was already excepting raw HTML.